prime-router/src/main/kotlin/azure/BlobAccess.kt

```diff
@@ -58,26 +58,26 @@ class BlobAccess(
         receivingApplicationReport: String? = null,
         receivingFacilityReport: String? = null
     ): BlobInfo {
-        val (bodyFormat, blobBytes) = createBodyBytes(
+        val blobBytes = createBodyBytes(
             report,
             sendingApplicationReport,
             receivingApplicationReport,
             receivingFacilityReport
         )
-        return uploadBody(bodyFormat, blobBytes, report.name, subfolderName, action)
+        return uploadBody(report.bodyFormat, blobBytes, report.name, subfolderName, action)
     }
 
     /**
      * Uses a serializer that matches the bodyFormat of the passed in [report] to generate a ByteArray to upload
      * to the blobstore. [sendingApplicationReport], [receivingApplicationReport], and [receivingFacilityReport] are
      * optional parameter that should be populated solely for empty HL7_BATCH files.
      */
-    private fun createBodyBytes(
+    fun createBodyBytes(
         report: Report,
         sendingApplicationReport: String? = null,
         receivingApplicationReport: String? = null,
         receivingFacilityReport: String? = null
-    ): Pair<Report.Format, ByteArray> {
+    ): ByteArray {
```

Comment:
Returning ByteArray instead of a Pair<ByteArea, Format> because it is more useful.